kiwi/schema/kiwi.rnc

```diff
@@ -37,7 +37,7 @@ vhd-tag-type = xsd:token {pattern = "[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}"}
 groups-list = xsd:token {pattern = "[a-zA-Z0-9_\-\.:]+(,[a-zA-Z0-9_\-\.:]+)*"}
 arch-name = xsd:token {pattern = "(x86_64|i586|i686|ix86|aarch64|arm64|armv5el|armv5tel|armv6hl|armv6l|armv7hl|armv7l|ppc|ppc64|ppc64le|s390|s390x|riscv64)(,(x86_64|i586|i686|ix86|aarch64|arm64|armv5el|armv5tel|armv6hl|armv6l|armv7hl|armv7l|ppc|ppc64|ppc64le|s390|s390x|riscv64))*"}
 portnum-type = xsd:token {pattern = "(\d+|\d+/(udp|tcp))"}
-grub_console = xsd:token {pattern = "(console|gfxterm|serial)( (console|gfxterm|serial))*"}
+grub_console = xsd:token {pattern = "(\s*|console|gfxterm|serial)( (console|gfxterm|serial))*"}
```

Comment:
Could we make this more explicit by providing a ```none``` console type ? Something like this

```
grub_console = xsd:token {pattern = "(none|console|gfxterm|serial)( (console|gfxterm|serial))*"}
```

It imho makes it more clear what is expected to happen with the console setting and could also be
found better in the documentation. 

Talking about the docs, could you add some words about this in
```doc/source/image_description/elements.rst```

Thanks much

Reply:
@leifliddy any thoughts ? Thanks

Reply:
Yup, that makes perfect sense. Sorry, I was on a motorcycle trip over the last several days. I'll try and make these changes tonight.

Reply:
excellent thanks much :+1: